src/xunit.analyzers/Descriptors.cs

```diff
@@ -254,7 +254,10 @@ internal static class Descriptors
             "Keep precision in range {0} when asserting equality of {1} typed actual value.",
             Categories.Assertions, DiagnosticSeverity.Error, isEnabledByDefault: true);
 
-        // Placeholder for rule X2017
+        internal static DiagnosticDescriptor X2017_AssertIsTypeShouldNotBeUsedForAbstractType { get; } = new DiagnosticDescriptor("xUnit2017",
```

Comment:
Sure. BTW, your project is the first I've seen that has a separate class for descriptors, rather than putting the descriptors in the analyzers. Have you considered the latter option?